### PR TITLE
Feature: Custom Subtitles, loaded from hard drive (#19)

### PR DESCRIPTION
**Pull request by ivantodorovich**
Feature: Add custom subtitles #19

Adds an item in the player's subtitle menu.

It also cleanups videojsplugins, needed to implement this functionality.

May need better error handling.

### CODE REVIEW

**Comment by GonzaGonza** [comment]
This looks promising! Let me test it! Thanks!!
Yes as you saw, yesterday @Patineta did some clean up with the files we had.
We still need some error handling in both methods (local srt and downloading a zip with subs)

**Comment by Patineta** [comment]
@ivantodorovich Loved the feature -and complementary refactor- (a ton of people have been requesting that feature for a while). Thanks a lot for this PR!

File: js/vendor/videojsplugins.js [hunk 1]
```diff
@@ -80,82 +130,119 @@ vjs.TextTrack.prototype.load = function(){
...
+
+        callback(dataBuff.toString('utf-8'));
+      }
+    }
+
+    // Get it, Unzip it, Decode it, Send it
+    var this_ = this;
+    getSub(this.src_, function(dataBuf) {
+      var path = require('path');
+      if (path.extname(this_.src) === '.zip') {
```
Comment (ivantodorovich) [inline js/vendor/videojsplugins.js:235]: Beware of bug. Line 233 should be:

```
if (path.extname(this_.src_) === '.zip') {
```

I saw @Patineta fixed this in another commit later on. (https://github.com/popcorn-time/popcorn-app/commit/10bba98cba5a2884d97b23a914ba1d8f6093bb48)

**Comment by EvandroZanatta** [comment]
:+1:

**Comment by GonzaGonza** [comment]
This PR is closed :( ... i will copy your changes and try it on a local branch.
If everything goes fine i will commit changes. (i will credit you in the comments @ivantodorovich )
This looks promising :+1:

**Comment by ivantodorovich** [comment]
That's ok. I thought It was already merged to master o_O (https://github.com/popcorn-time/popcorn-app/commit/096bd57f87026a594a4642a6d4ef83a42b273209 by @Patineta)
Am I missing something?

**Comment by GonzaGonza** [comment]
Lol, yes its pushed by @Patineta . Thanks @ivantodorovich . Maybe i need some sleep :+1:

**Pull request closed without merging**